homeassistant/components/freebox/home_base.py

```diff
@@ -0,0 +1,129 @@
+            if node["type"].get("inherit") == "node::rts":
+                self._manufacturer = "Somfy"
+                self._model = CATEGORY_TO_MODEL.get("rts")
+            elif node["type"].get("inherit") == "node::ios":
+                self._manufacturer = "Somfy"
+                self._model = CATEGORY_TO_MODEL.get("iohome")
+
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, self._id)},
+            name=self._device_name,
+            manufacturer=self._manufacturer,
+            model=self._model,
+            sw_version=self._firmware,
+        )
+
+    async def async_update_signal(self):
+        """Update signal."""
+        self._node = self._router.home_devices[self._id]
+        # Update name
+        if self._sub_node is None:
+            self._attr_name = self._node["label"].strip()
+        else:
+            self._attr_name = (
+                self._node["label"].strip() + " " + self._sub_node["label"].strip()
+            )
+        self.async_write_ha_state()
+
+    async def set_home_endpoint_value(self, command_id: Any, value=None) -> None:
+        """Set Home endpoint value."""
+        if value is None:
+            value = {"value": None}
+        if command_id is None:
+            _LOGGER.error("Unable to SET a value through the API. Command is None")
+            return
+        await self._router.home.set_home_endpoint_value(
+            self._id, command_id, {"value": value}
+        )
+
+    def get_command_id(self, nodes, name) -> int | None:
+        """Get the command id."""
+        node = next(
+            filter(lambda x: (x["name"] == name), nodes),
+            None,
+        )
+        if node is None:
+            _LOGGER.warning("The Freebox Home device has no value for: %s", name)
+            return None
+        return node["id"]
+
+    async def async_added_to_hass(self):
+        """Register state update callback."""
+        self.remove_signal_update(
+            async_dispatcher_connect(
+                self._hass,
+                self._router.signal_home_device_update,
+                self.async_update_signal,
+            )
+        )
+
+    async def async_will_remove_from_hass(self):
+        """When entity will be removed from hass."""
+        self._remove_signal_update()
+
+    def remove_signal_update(self, dispacher: None):
+        """Register state update callback."""
+        self._remove_signal_update = dispacher
+
+    def get_value(self, ep_type, name):
+        """Get the value."""
+        node = next(
+            filter(
+                lambda x: (x["name"] == name and x["ep_type"] == ep_type),
+                self._node["show_endpoints"],
+            ),
+            None,
+        )
+        if node is None:
+            _LOGGER.warning(
+                "The Freebox Home device has no node for: " + ep_type + "/" + name
+            )
+            return VALUE_NOT_SET
+        return node.get("value", VALUE_NOT_SET)
```

Comment:
```suggestion
            return None
        return node.get("value")
```
We already told that `VALUE_NOT_SET` should not exist.